### PR TITLE
support for await...of

### PR DESCRIPTION
**Pull request by YuichiNukiyama**
### repated pages
- https://github.com/tc39/proposal-async-iteration
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of

### CODE REVIEW

File: javascript/statements.json [hunk 1]
```diff
@@ -964,6 +964,61 @@
...
+      "for_await_of": {
+        "__compat": {
+          "description": "<code>for await...of</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/for-await...of",
+          "support": {
+            "chrome": {
+              "version_added": "63"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline javascript/statements.json]: This should match Chrome, as should `webview_android`.

Please wait to change this. I want to confirm the version number.

Comment (jpmedley) [reply javascript/statements.json]: Confirmed. Please change.

**Comment by YuichiNukiyama** [comment]
Thanks for your review
I add Opera's support, too.
https://www.chromestatus.com/feature/5727385018171392

File: javascript/statements.json [hunk 2]
```diff
@@ -964,6 +964,61 @@
...
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": "63"
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
```
Comment (ddbeck) [inline javascript/statements.json]: This looks to be on the [standards track](https://tc39.github.io/ecma262/#sec-for-in-and-for-of-statements).

```suggestion
            "standard_track": true,
```

**Comment by YuichiNukiyama** [comment]
:smile:

**Comment by ddbeck** [comment]
Thank you, @YuichiNukiyama! 🎉

**Pull request merged**